index.html

```diff
@@ -14,23 +14,157 @@
+                <a href="#" class="logo"><span>My Logo</span></a>
+                <div class="menu">
+                    <a href="#" class="desktop-link">Portfolio</a>
+                    <a href="#" class="desktop-link">About</a>
+                    <a href="#" class="desktop-link">Contact</a>
+                </div>
+                <a href="#" class="mobile-link menu"><i class="fas fa-bars"></i></a>
             </nav>
         </header>
-    
+        
         <div class="headline">
-            <h1>I'm Fikerte Glad to see you!</h1>
-            <p>I’m a software developer! I can help you build a product ,
+            <div class="headline-text-container">
+                <h1>I'm Fikerte Glad to see you!</h1>
+                <p>I’m a software developer! I can help you build a product ,
+                    feature or website Look through some of my work and experience!
+                    If you like what you see and have a project you need coded,
+                    don’t hestiate to contact me.</p>
+                <a href="#">Let's connect</a>
+                <ul class="social-icons">
+                    <li>
+                        <a href="#"><i class="fab fa-twitter"></i></a>
+                    </li>
+                    <li>
+                        <a href="#"><i class="fab fa-linkedin-in"></i></a>
+                    </li>
+                    <li>
+                        <a href="#"><i class="fab fa-medium-m"></i></a>
+                    </li>
+                    <li>
+                        <a href="#"><i class="fab fa-github"></i></a>
+                    </li>
+                    <li>
+                        <a href="#"><i class="fab fa-angellist"></i></a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </section>
+
+    <section class="card-works">
+        <section class="card card-1">
+            <div class="card-img-1 card-img"></div>
+            <div class="card-text">
+                <h2>Tonic</h2>
+                <div class="title-role">
+                    <ul>
+                        <li>CANOPY</li>
+                        <li class="role"><i class="fas fa-circle"></i></li>
+                        <li class="role">Back End Dev</li>
+                        <li class="role"><i class="fas fa-circle"></i></li>
+                        <li class="role">2015</li>
+                    </ul>
+                </div>        
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="languages">
+                    <ul>
+                        <li><a href="#">html</a></li>
+                        <li><a href="#">css</a></li>
+                        <li><a href="#">javascript</a></li>
+                    </ul>
+                </div>
+                <button class="button" type="button">See Project</button>
+            </div>
+        </section>
```

Comment:
- [ ] Can you please give equal with to the **project image** and **project details**, for now, the image seems to take only `40%` of the width and the project details `50%` that's why the image is not well aligned compared to the Figma design, see the difference below 👇 

- Your design
![image](https://user-images.githubusercontent.com/55465133/130153672-a128949a-d7ec-4316-a77e-0674c626d7dc.png)

- Figma design
![image](https://user-images.githubusercontent.com/55465133/130153709-a4fdb7eb-b87d-4a5a-80c2-51219393127d.png)

You can see that your image is stretched compared to the one in Figma, try to give equal with to the image and the details (50%) both, this should fix the issue. You can do the same for all the projects 😉